### PR TITLE
mark elm-test as unmaintained

### PR DESCRIPTION
**Pull request by harrysarson**
Also adds a note explaining the move to elm-explorations

### CODE REVIEW

File: maintainers.md [hunk 1]
```diff
@@ -48,3 +47,4 @@ This file contains info on elm-community packages and who the current maintainer
 | [elm-history](http://github.com/elm-community/elm-history) | Elm bindings to HTML5 History API | *Unmaintained* |  |
 | [elm-lang.org](http://github.com/elm-community/elm-lang.org) | The full source for http://elm-lang.org/, the home-page of the Elm programming language. Open sourced as a way to teach people how to write and serve Elm code. Follow the instructions in README.md to get the site setup on your own machine. | *Unmaintained* |  |
```
Comment (prikhi) [inline maintainers.md:48]:
```suggestion
| [elm-lang.org](http://github.com/elm-community/elm-lang.org) | The full source for http://elm-lang.org/, the home-page of the Elm programming language. Open sourced as a way to teach people how to write and serve Elm code. Follow the instructions in README.md to get the site setup on your own machine. | *Unmaintained* |  |
| [elm-test](http://github.com/elm-community/elm-test) | The 0.18.0 version of [elm-explorations/test](http://github.com/elm-explorations/test) | *Unmaintained* | |
```

File: maintainers.md [hunk 2]
```diff
@@ -48,3 +47,4 @@ This file contains info on elm-community packages and who the current maintainer
...
 | [html-test-runner](http://github.com/elm-community/html-test-runner) | Run elm-test suites in the browser | *Unmaintained* |  |
+| [elm-test](http://github.com/elm-community/elm-test) | The 0.18.0 version of [elm-explorations/test](http://github.com/elm-explorations/test) | *Unmaintained* | |
```
Comment (prikhi) [inline maintainers.md:49]:
```suggestion
```

**Review by prikhi — COMMENTED** [review]
Keep it alphabetical?

**Comment by harrysarson** [comment]
Thanks @prikhi, I have added the changes :)

**Comment by prikhi** [comment]
Thanks!

**Pull request merged**